.github/workflows/nfr.yml

```diff
@@ -35,104 +38,142 @@ permissions:
-      working-directory: ./tests
-      run:
-        make create-gke-cluster CI=true
-
-    - name: Create and setup VM
-      working-directory: ./tests
-      run:
-        make create-and-setup-vm
-
-    - name: Run Tests
-      working-directory: ./tests
-      run: |
-        if ${{ inputs.test_label != 'all' }}; then
-        sed -i '/^GINKGO_LABEL=/s/=.*/="${{ inputs.test_label }}"/' "scripts/vars.env" && make nfr-test;
-        else
-        make nfr-test;
-        fi
-
-    - name: Cleanup
-      working-directory: ./tests
-      if: always()
-      run: |
-        bash scripts/cleanup-vm.sh true
-        make delete-gke-cluster
-        rm -rf scripts/vars.env
-
-    - name: Open a PR with the results
-      uses: peter-evans/create-pull-request@c5a7806660adbe173f04e3e038b0ccdcd758773c # v6.1.0
-      with:
-        token: ${{ secrets.NGINX_PAT }}
-        commit-message: NFR Test Results for NGF version ${{ inputs.version }} ${{ inputs.nginx_plus == true && '(Plus)' || ''}}
-        author: nginx-bot <integrations@nginx.com>
-        committer: nginx-bot <integrations@nginx.com>
-        branch: tests/nfr-tests-${{ inputs.version }}${{ inputs.nginx_plus == true && '-plus' || ''}}
-        delete-branch: true
-        title: NFR Test Results for NGF version ${{ inputs.version }} ${{ inputs.nginx_plus == true && '(Plus)' || ''}}
-        add-paths: |
-          tests/results/
-        body: |
-          Update with NFR test results for NGF version ${{ inputs.version }} ${{ inputs.nginx_plus == true && '(Plus)' || ''}}
-          - Auto-generated by the NFR tests workflow run ${{ github.run_id }}
-          - Tests ran using Docker image tag ${{ inputs.image_tag }}
-          - ${{ inputs.test_label }} test(s) ran
-          - NGINX Plus enabled: ${{ inputs.nginx_plus }}
-        labels: |
-          tests
-        assignees: ${{ github.actor }}
-        draft: true
+      - name: Checkout Repository
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+
+      - name: Download Artifacts
+        uses: actions/download-artifact@fa0a91b85d4f404e444e00e005971372dc801d16 # v4.1.8
+        with:
+          path: tests/results/
+
+      - name: Open a PR with the results
+        uses: peter-evans/create-pull-request@c5a7806660adbe173f04e3e038b0ccdcd758773c # v6.1.0
+        with:
+          token: ${{ secrets.NGINX_PAT }}
+          commit-message: NFR Test Results for NGF version ${{ needs.vars.outputs.version }}
+          author: nginx-bot <integrations@nginx.com>
+          committer: nginx-bot <integrations@nginx.com>
+          branch: tests/nfr-tests-${{ needs.vars.outputs.version }}
+          delete-branch: true
+          title: NFR Test Results for NGF version ${{ needs.vars.outputs.version }}
+          add-paths: |
+            tests/results/
+          body: |
+            Update with NFR test results for NGF version ${{ needs.vars.outputs.version }} ${{ needs.vars.outputs.types }}
```

Comment:
${{ needs.vars.outputs.types }} will just say "both" by default, which may not make sense in the description.

Reply:
this will say `["oss", "plus"]`...it's kinda ugly but not sure if we really need to clean it up 😅

Reply:
Oh ok, I'm fine with that.